aws/table_aws_ssm_association.go

```diff
@@ -0,0 +1,204 @@
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "compliance_severity",
+				Description: "A cron expression that specifies a schedule when the association runs.",
+				Hydrate:     getAwsSSMAssociation,
+				Type:        proto.ColumnType_STRING,
+			},
+			/// Standard columns for all tables
+			{
+				Name:        "title",
+				Description: resourceInterfaceDescription("title"),
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Name"),
+			},
+			{
+				Name:        "akas",
+				Description: resourceInterfaceDescription("akas"),
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getAwsSSMAssociationAkas,
+				Transform:   transform.FromValue(),
+			},
+		}),
+	}
+}
+
+//// LIST FUNCTION
+
+func listAwsSSMAssociations(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
+	// TODO put me in helper function
+	var region string
+	matrixRegion := plugin.GetMatrixItem(ctx)[matrixKeyRegion]
+	if matrixRegion != nil {
+		region = matrixRegion.(string)
+	}
+	plugin.Logger(ctx).Trace("listAwsSSMAssociations", "AWS_REGION", region)
+
+	// Create session
+	svc, err := SsmService(ctx, d, region)
+	if err != nil {
+		return nil, err
+	}
+
+	// List call
+	err = svc.ListAssociationsPages(
+		&ssm.ListAssociationsInput{},
+		func(page *ssm.ListAssociationsOutput, isLast bool) bool {
+			for _, association := range page.Associations {
+				d.StreamListItem(ctx, &ssm.AssociationDescription{
+					Name:               association.Name,
+					AssociationId:      association.AssociationId,
+					AssociationName:    association.AssociationName,
+					AssociationVersion: association.AssociationVersion,
+					Targets:            association.Targets,
+					LastExecutionDate:  association.LastExecutionDate,
+					Overview:           association.Overview,
+				})
+
+			}
+			return !isLast
+		},
+	)
+
+	return nil, err
+}
+
+// HYDRATE FUNCTIONS
+//TODO
+
+func getAwsSSMAssociation(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	logger := plugin.Logger(ctx)
+	logger.Trace("getAwsSSMAssociation")
+
+	// TODO put me in helper function
+	var region string
+	matrixRegion := plugin.GetMatrixItem(ctx)[matrixKeyRegion]
+	if matrixRegion != nil {
+		region = matrixRegion.(string)
+	}
+	var id string
+	if h.Item != nil {
+		id = *h.Item.(*ssm.AssociationDescription).AssociationId
+	} else {
+		id = d.KeyColumnQuals["association_id"].GetStringValue()
+	}
+
+	// Create Session
+	svc, err := SsmService(ctx, d, region)
+	if err != nil {
+		return nil, err
+	}
+
+	// Build the params
+	params := &ssm.DescribeAssociationInput{
+		AssociationId: aws.String(id),
+	}
+
+	// Get call
+	data, err := svc.DescribeAssociation(params)
+	if err != nil {
+		logger.Debug("getAwsSSMAssociation", "ERROR", err)
+		return nil, err
+	}
+	return data.AssociationDescription, nil
```

Comment:
Can we return `data`, and handle the same when we are defining the column (i.e. `transform.FromField()`)?

Reply:
@Priyanka585464 Was this change made if it's still applicable?

Reply:
Not required according to @Subhajit97 , resolving.